PrimeNumberTheoremAnd/ZetaBounds.lean

```diff
@@ -57,13 +58,33 @@ theorem riemannZetaResidue :
 \end{proof}
 %%-/
 
+/- New two theorems to be proven -/
+
+theorem analytic_deriv_bounded_near_point
+  (f : ℂ → ℂ) {U : Set ℂ} {p : ℂ} (hU : IsOpen U) (hp : p ∈ U) (hf : HolomorphicOn f U) :
+  (deriv f) =O[𝓝[≠] p] (1 : ℂ → ℂ ) := by
+    --refine (analyticOn_iff_differentiableOn hU).mp ?_
+    sorry
+
+
+theorem analytic_bounded_near_point
+  (f : ℂ → ℂ) {U : Set ℂ} {p : ℂ} (hU : IsOpen U) (hp : p ∈ U) (hf : HolomorphicOn f U) :
+  f =O[𝓝[≠] p] (1 : ℂ → ℂ ) := by
+    --refine (analyticOn_iff_differentiableOn hU).mp ?_
+    sorry
+
 /-%%
 \begin{theorem}[logDerivResidue]\label{logDerivResidue}\lean{logDerivResidue}\leanok
   If $f$ is holomorphic in a neighborhood of $p$, and there is a simple pole at $p$, then $f'/f$ has a simple pole at $p$ with residue $-1$:
   $$ \frac{f'(s)}{f(s)} = \frac{-1}{s - p} + O(1).$$
 \end{theorem}
 %%-/
-theorem logDerivResidue {f : ℂ → ℂ} {p : ℂ} {U : Set ℂ} (holc : HolomorphicOn f (U \ {p}))
+
+/- The set should be open so that f'(p) = O(1) for all p ∈ U -/
+
+theorem logDerivResidue {f : ℂ → ℂ} {p : ℂ} {U : Set ℂ}
+    (U_is_open : IsOpen U)
```

Comment:
You should be able to get something like this from `U_in-nhds` -- this will give you that there is an open set contained in `U` which contains `p`...